canonical-web-and-design/ubuntu.com
# New /kernel/lifecycle page

## Done

- Created a new /kernel/lifecycle page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/kernel/lifecycle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare the text to the [copy doc](https://docs.google.com/document/d/1d01SFs7c56ge2o92g6Rwr3SjG8rtxm3jNwIyL1N8mQg/edit#heading=h.3gbb9wcbk9us) and the charts to the [sheet](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=703230516)

## TODO

- [x] Possibly put the charts in a menu like the instructions
- [x] Possibly reverse the order of versions to match the instructions
- [ ] Get the data for 18.04 and in general confirmed by Lech and Brad

## Issue / Card

Fixes #5684
